utils/pswdgen/generator.go

```diff
@@ -139,11 +139,17 @@ func Generate(config Config) (string, error) {
 
 func readTermChar() ([]byte, error) {
 	t, _ := term.Open("/dev/tty")
-	term.RawMode(t)
+	if err := term.RawMode(t); err != nil {
+		return nil, err
+	}
 	bytes := make([]byte, 3)
 	numRead, err := t.Read(bytes)
```

Comment:
Please add an error check for `t.Read(bytes)`

Reply:
Good catch, fixed.